.gitlab-ci.yml

```diff
@@ -30,6 +30,27 @@ test-features:
   script:
     - cargo test --all --all-features --all-targets --locked
 
+integration-tests:
+  timeout: 5 minutes
+  image: paritytech/ci-unified:bullseye-1.70.0
+  stage: test
+  script:
+    - curl -fsSL https://deb.nodesource.com/setup_20.x | bash -
+    - apt-get install -y nodejs
+    - cargo build -p spiritnet-runtime
+    - cd ./integration-tests/chopsticks
+    - yarn 
+    - yarn test
+
+lint-typescript:
```

Comment:
I don't think we need a whole separate job to check for lints. We can add it to the previous step.

Reply:
I am not in favor of mixing these two tests into one. Checking for quality, such as formatting and linting, should, in my opinion, be a single step. The second step should only focus on the logic itself, I would say.

Reply:
Yeah but these are integration tests, we are not publishing a library. I think it's overkill to take CI time to test for something that it's "nice to have" but not strictly required. What would be the downside of having the lints checked before the tests are run? If the tests pass but there's a lint issue, the next push will trigger the whole thing again, and we have just "wasted" time to re-run tests that already passed.

Reply:
Yeah okay. The tests is actually super fast (takes less then 1 min). But I added the check a step ahead: [63a0d0c](https://github.com/KILTprotocol/kilt-node/pull/614/commits/63a0d0cd7ca9f852e8802b58234b9877b6bf164e)